nexusadspy/client.py

```diff
@@ -117,6 +118,9 @@ def _do_paged_get(self, url, method, params=None, data=None, headers=None,
             if max_items is not None and len(res) >= max_items:
```

Comment:
The response in /siphon API can range from 1 entry to thousands. So `max_items` would not help here.